### DISCUSSION TITLE
Include support for K3S at https://github.com/k3s-io/k3s

### DISCUSSION THREAD

**Discussion by nathanawmk** [linkerd/linkerd2#7240] [Q&A]
I wish to implement MTLS for K3S. Is it in the Linkerd roadmap to include support for K3S? 

https://github.com/k3s-io/k3s
https://github.com/k3s-io/k3s/issues/4417
https://nathanawmk.github.io/

**Comment by kleimkuhler** [comment]
@nathanawmk If I'm reading your question correctly, I moved this to a discussion as this is not an issue with Linkerd, but more how to install it on a k3s cluster.

Have you looked at [k3d](https://github.com/rancher/k3d)? You can run a containerized k3s cluster and install Linkerd without issue; most of the Linkerd developers use this as does Linked's CI.

If you have a k3s cluster without k3d, that should also be fine. Have you run into an issue installing Linkerd on a k3s cluster?
